core/blockchain.go

```diff
@@ -1712,6 +1721,7 @@ func (bc *BlockChain) insertChain(chain types.Blocks, verifySeals bool) (int, er
 
 		if dirtyAccounts != nil {
 			bc.dirtyAccountFeed.Send(dirtyAccounts)
+			bc.dirtyAccountsCache.Add(block.Hash(), dirtyAccounts)
```

Comment:
I think it's better to cache dirty accounts when a configuration is on only, not to always do this.

Reply:
I think dirty accounts cache is not big compare to other caches such as block. Therefore, we don't need to add more configuration for it